xml/security_cryptofs.xml

```diff
@@ -269,4 +278,83 @@
     the file.
   </para>
  </sect1>
+ <sect1 xml:id="sec-security-cryptofs-rage">
+  <title>Encrypting files with Rage</title>
+  <para>
+   Rage is a secure file encryption software to encrypt files.
+   Rage has small explicit keys and does not have configuration options.
+  </para>
+  <para>You must first generate a key pair to encrypt a file with Rage:
+   </para>
+   <screen>&prompt.user;rage-keygen -o ~/rage.key 2 ~/rage.pub</screen>
+   <para>Note that two files are created; <emphasis>rage.pub</emphasis> and <emphasis>rage.key</emphasis>.
+   A recipient can be either an <emphasis>age</emphasis> or <emphasis>SSH</emphasis> public key.</para>
```

Comment:
ssh rsa key or ssh ed25519 key specifically.

Reply:
given you have an ssh section later, you likely don't need this comment about ssh keys here.

Reply:
Yeah , good point , removed this line.